### ISSUE TITLE
Can you describe working of  serverside  ?

### ISSUE THREAD

**Issue by ghost** [zhangela/handipark#1]
What I have to do if I want to use ?. Do I just install app?. or I have to configure server side too ?.

**Comment by sasukeourad** [comment]
Yep .. you'll need to install the app and configure the server too.

Sent from my NOKIA 3310

----- Reply message -----

**Comment by ghost** [comment]
How do I that? If you explain a little bit I'm thankful.
